solidity/test/integration/data/bls.ts

```diff
@@ -0,0 +1,10 @@
+const blsData = {
+  // data generated using master secret key 123
+  secretKey: 123,
+
+  // altbn128 public key for secret key 123. For implementation details refer to bls_test.go
+  groupPubKey:
+    "0x1f1954b33144db2b5c90da089e8bde287ec7089d5d6433f3b6becaefdb678b1b2a9de38d14bef2cf9afc3c698a4211fa7ada7b4f036a2dfef0dc122b423259d01659dc18b57722ecf6a4beb4d04dfe780a660c4c3bb2b165ab8486114c464c621bf37ecdba226629c20908c7f475c5b3a7628ce26d696436eab0b0148034dfcd",
```

Comment:
Could we split the hex-string into multiple lines for easier reading?
```suggestion
     "0x1f1954b33144db2b5c90da089e8bde287ec7089d5d6433f3b6becaefdb678b1b2a9de3" +
    "8d14bef2cf9afc3c698a4211fa7ada7b4f036a2dfef0dc122b423259d01659dc18b57722" +
    "ecf6a4beb4d04dfe780a660c4c3bb2b165ab8486114c464c621bf37ecdba226629c20908" +
    "c7f475c5b3a7628ce26d696436eab0b0148034dfcd",
    ```